modules/gateways/callback/blockonomics.php

```diff
@@ -75,9 +75,9 @@
 $underpayment_slack = $blockonomics->getUnderpaymentSlack() / 100 * $bits;
 if ($value < $bits - $underpayment_slack) {
     $price_by_expected = $blockonomics->getPriceByExpected($invoiceId);
-    $paymentAmount = round($paid * $price_by_expected, 2);
+    $paymentAmount = round($paid * $price_by_expected, 5);
```

Comment:
About the precision here. Look at why this is important, in this example (using real currency VES)
The amount of the transaction added into the invoice will be:

72.944,00 with precision 5
72.000,00 with precision 2

Which makes a big difference, in fact unfairly disadvantaging the customer.